src/components/MoneyReportHeader.tsx

```diff
@@ -143,6 +144,8 @@ function MoneyReportHeader({policy, report: moneyRequestReport, transactionThrea
 
     const shouldShowSubmitButton =
         !!moneyRequestReport &&
+        !isArchivedReport &&
```

Comment:
I don't believe we need `isArchivedReport` here, `isArchivedExpenseReport` should suffice since only expense type reports will have action buttons.